CMakeLists.txt

```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 message(STATUS "Building using CMake version: ${CMAKE_VERSION}")
-cmake_minimum_required(VERSION 3.14)
+cmake_minimum_required(VERSION 3.22)
```

Comment:
The version bump is required to support the DEPENDS condition in the dependent option:

https://cmake.org/cmake/help/v3.29/module/CMakeDependentOption.html

Might be a creative way to work around this if 3.14 is a hard requirement